docs/src/hugo/content/introduction/competitors.md

```diff
@@ -2,30 +2,24 @@
-#    parent: 'intro'
-#    identifier: intro-competitors
-#    url: /introduction/competitors.html
-#    weight: 3
+menu:
+  main:
+    parent: 'intro'
+    identifier: intro-competitors
+    url: /introduction/competitors.html
+    weight: 3
 ---
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas eu commodo nisi. Etiam sagittis enim purus, id tristique ante tincidunt id. Nunc placerat velit neque. Integer finibus velit nec vestibulum elementum. Mauris vel tempor velit. Nam sodales malesuada purus vel bibendum. Sed eu mauris sit amet nulla sodales imperdiet. Sed pulvinar iaculis magna at feugiat. Quisque luctus nec lectus vitae pharetra. Cras euismod pharetra augue ac consequat. Integer id euismod lectus, non cursus ligula. Nunc lorem dolor, condimentum nec nunc non, hendrerit ullamcorper tellus. Sed tortor nibh, mollis eget euismod ac, tempor a eros. Etiam feugiat odio ac interdum pellentesque. Nam dignissim, ipsum ut vehicula interdum, justo dui accumsan felis, in maximus diam justo tristique tellus.
-
 ## Spinnaker
 
-YYYY Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas eu commodo nisi. Etiam sagittis enim purus, id tristique ante tincidunt id. Nunc placerat velit neque. Integer finibus velit nec vestibulum elementum. Mauris vel tempor velit. Nam sodales malesuada purus vel bibendum. Sed eu mauris sit amet nulla sodales imperdiet. Sed pulvinar iaculis magna at feugiat. Quisque luctus nec lectus vitae pharetra. Cras euismod pharetra augue ac consequat. Integer id euismod lectus, non cursus ligula. Nunc lorem dolor, condimentum nec nunc non, hendrerit ullamcorper tellus. Sed tortor nibh, mollis eget euismod ac, tempor a eros. Etiam feugiat odio ac interdum pellentesque. Nam dignissim, ipsum ut vehicula interdum, justo dui accumsan felis, in maximus diam justo tristique tellus.
-
-## Collins
+One of the most common systems Nelson is compared to is the [Spinnaker project](https://spinnaker.io). On the face of it, the systems are indeed solving similar problems. The truth is however a little more nuanced. Spinnaker was born out of Netflix as a replacement for their [Asguard deployment system](https://github.com/Netflix/asgard). The primary difference between Spinnaker and Nelson is that Nelson is container-first where as Spinnaker was primarily designed to operate with Netflix's virtual machine "bakery". Whilst they have since retrofitted support for containers, the approach of the project cannot detech itself from that legacy workflow easily. Given Nelson is focused on integration with scheduling systems as its common substrate for deployment, the management of the virtual machines underlying those schedulers is out of scope. Indeed, with the prevelance of hosted scheudling systems like [EKS](https://aws.amazon.com/eks/), [EKS](https://cloud.google.com/kubernetes-engine/) and [AKE](https://azure.microsoft.com/en-us/services/kubernetes-service/), the need for even running your own machines appears to be passing.
```

Comment:
Took a look, I'll rephrase some parts of this (in flight tomorrow) and send it over when I land.